src/test/java/org/isf/patient/rest/PatientControllerTest.java

```diff
@@ -343,7 +343,7 @@ public void whet_get_patients_non_parameters_then_return_list_of_PatientDTO_page
 
 		List<PatientDTO> expectedPatientDTOList = patientMapper.map2DTOList(patientList);
 
-		when(patientBrowserManagerMock.getPatient(any(Integer.class), any(Integer.class)))
+		when(patientBrowserManagerMock.getPatient(any(int.class), any(int.class)))
```

Comment:
is this change consistend? `int.class`?

Reply:
Yes; it works but if you find it bad I've changed it to `anyInt()'